README.md

```diff
@@ -18,14 +18,22 @@ cph approve --name pipeline_name
 1. Clone the repo:
 
     `git clone https://github.com/shreyasrama/cph.git`
+    
+1. Change directories:
+
+    `cd cph`
 
 1. Build a binary:
 
     `go build -mod=mod -o bin/cph main.go`
+    
+1. Move to PATH:
+
```

Comment:
Want to rewrite this to use Logan's suggestion of `go install github.com/shreyasrama/cph@latest`?